di-ipv-ticf-stub/openAPI/ticf-external.yaml

```diff
@@ -83,3 +113,15 @@ components:
           items:
             type: string
           description: An array of VC's from CRI's
+    UserEvidenceRequestBody:
+      description: Request body for Ticf management
+      type: object
+      properties:
+        userId:
```

Comment:
I think this is a path parameter, rather than being in the request body - the body just contains the evidence I think?

Reply:
done